src/pykeen/nn/emb.py

```diff
@@ -103,12 +123,51 @@ def __init__(
             embedding_dim=embedding_dim,
         )
 
+    @classmethod
+    def from_specification(
```

Comment:
the instantiation of an embedding from the specification should be a member function of the EmbeddingSpecification class

Reply:
You mean something like `EmbeddingSpecification.create(num, dim)` ?

Reply:
done in 
0ada540  ?

Reply:
yup